doc/entity.md

```diff
@@ -0,0 +1,117 @@
+# Entity
+
+An entity is the way you describe objects you want to work with in wetland, wetland will create the sql schema for you.
```

Comment:
Entities are objects with a conceptual identity assigned within your domain. They're regular classes that hold a state you describe. Entities are mapped when registered with the entity manager, allowing you to describe what the entity looks like on the persisted side (the database).

In short, entities are simple classes that describe tables in your database, of which instances hold state in your application and map to specific rows in your database.